ram/hram.asm

```diff
@@ -85,10 +85,10 @@ hSpriteScreenX:: db
 hSpriteScreenY:: db
 
 NEXTU
-hFF8F:: db
-hFF90:: db
-hFF91:: db
-hFF92:: db
+hSpriteCollisionsLoopCounter:: db
+hSpriteCollisionsTempYCoord:: db
```

Comment:
First this is initialized to `[i#SPRITESTATEDATA1_YPIXELS]` with some +/-7 adjustment, and stored in `[i#SPRITESTATEDATA1_YADJUSTED]`. Then the absolute difference between `[i#SPRITESTATEDATA1_YADJUSTED]` and `[j#SPRITESTATEDATA1_YADJUSTED]` is stored in it , and used to calculate `hSpriteCollisionsAdjustedDistance`. It also has a "7 or 9 depending on sprite i's delta Y". I think it has enough overlapping purposes in the collision code that it can be thought of as a generic "Y-related temp value".
```suggestion
hCollidingSpriteTempYValue:: db
```